diff --git a/haskell/private/cc_wrapper_windows.sh.tpl b/haskell/private/cc_wrapper_windows.sh.tpl
index aa5613ec..980447c1 100644
--- a/haskell/private/cc_wrapper_windows.sh.tpl
+++ b/haskell/private/cc_wrapper_windows.sh.tpl
@@ -209,6 +209,8 @@ done
 # ----------------------------------------------------------
 # Call compiler
 
+echo "RESPONSE_FILE: $RESPONSE_FILE"
+
 "$CC" "@$RESPONSE_FILE"
 
 # vim: ft=sh
